### ISSUE TITLE
About training

### ISSUE THREAD

**Issue by dmmSJTU** [TencentARC/T2I-Adapter#111]
Hello ChongMou,

May I ask why you use “accelerate launch train_sketch.py --pretrained_model_name_or_path stabilityai/stable-diffusion-xl-base-1.0 --output_dir experiments/adapter_sketch_xl --config configs/train/Adapter-XL-sketch.yaml --mixed_precision="fp16" --resolution=1024 --learning_rate=1e-5 --max_train_steps=60000 --train_batch_size=1 --gradient_accumulation_steps=4 --report_to="wandb" --seed=42 --num_train_epochs 100” for training and report this problem?
<img width="876" alt="Snipaste_2023-12-26_14-28-39" src="https://github.com/TencentARC/T2I-Adapter/assets/111695222/bb981c01-a836-4f39-ad65-96899b727a5c">

Look forward for your reply.

**Comment by MC-E** [comment]
It seems that the training data (LAION) is incorrect. You can refer to https://github.com/TencentARC/T2I-Adapter/issues/99
